asciidoc-confluence-publisher-client/src/main/java/org/sahli/asciidoc/confluence/publisher/client/http/ConfluenceRestClient.java

```diff
@@ -357,6 +352,16 @@ private static ConfluenceAttachment extractConfluenceAttachment(JsonNode jsonNod
         return new ConfluenceAttachment(id, title, relativeDownloadLink, version);
     }
 
+    private static String extractAncestorIdFromJsonNode(JsonNode jsonNode) {
+        String ancestorId = null;
+        // last item in ancestors array if actual ancestor
+        Iterator<JsonNode> it = jsonNode.get("ancestors").elements();
```

Comment:
It appears counter-intuitive to me that a page in a hierarchy might have more than one ancestor - do you know any scenario in which that might me the case?